libvirt.py

```diff
@@ -27,6 +27,7 @@ def configure(self) -> None:
         self.hostconn.run_or_die("systemctl disable libvirtd.service")
         self._run_per_suffix("systemctl disable", "libvirtd", MONOLITHIC_SOCKET_SUFFIXES)
 
+        self.hostconn.run_or_die("systemctl restart virtqemud.service")
```

Comment:
I wonder why this would be needed, the sequence for configure is taken directly from https://libvirt.org/daemons.html#switching-to-modular-daemons